src/cpp/rtps/writer/BaseWriter.hpp

```diff
@@ -67,68 +67,31 @@ class BaseWriter
 
 public:
 
-    FASTDDS_EXPORTED_API bool has_been_fully_delivered(
-            const SequenceNumber_t& seq_num) const override;
+    //vvvvvvvvvvvvvvvvvvvvv [Exported API] vvvvvvvvvvvvvvvvvvvvv
 
-    FASTDDS_EXPORTED_API bool is_acked_by_all(
-            const SequenceNumber_t& seq_num) const override;
+    WriterListener* get_listener() const final;
 
-    FASTDDS_EXPORTED_API bool wait_for_all_acked(
-            const Duration_t& max_wait) override;
+    bool set_listener(
+            WriterListener* listener) final;
```

Comment:
This way, if the base function ceases to exist or it's made non-virtual we won't compile. Please apply to all inherited member functions. Applies to other files
```suggestion
            WriterListener* listener) override final;
```

Reply:
We decided that since the functions are not marked virtual on the derived class we don't need it.